docs/vi/week01/01-1.md

```diff
@@ -0,0 +1,94 @@
+---
+lang-ref: ch.01-1
+lecturer: Yann LeCun
+title: Động lực của học sâu, lịch sử và cảm hứng của nó
+authors: Yunya Wang, SunJoo Park, Mark Estudillo, Justin Mae
+date: 27 Jan 2020
+translator: Huynh Nguyen
+translator date: 01 Oct 2020
+---
+
+
+## [Nội dung](https://www.youtube.com/watch?v=0bMe_vCZo30&t=217s)
+
+- Kiến thức cơ bản của học có giám sát (Supervised Learning), mạng thần kinh (Neural Nets) và học sâu (Deep Learning)
+- Backpropagation and architectural components
+- Convolutional neural network and its applications
+- More Deep Learning Architectures
+- Regularization Tricks / Optimization Tricks / Understanding how Deep Learning works
+- Energy-based models
+- Self-supervised learning and beyond
+
+
+## Inspiration of Deep Learning and its history
+
+On a conceptual level, deep learning is inspired by the brain but not all of the brain's details are relevant. For a comparison, aeroplanes were inspired by birds. The principle of flying is the same but the details are extremely different.
+
+The history of deep learning goes back to a field which changed its name now to cybernetics. It started in the 1940s with McCulloch and Pitts. They came up with the idea that neurons are threshold units with on and off states. You could build a Boolean circuit by connecting neurons with each other and conduct logical inference with neurons. The brain is basically a logical inference machine because neurons are binary. Neurons compute a weighted sum of inputs and compare that sum to its threshold. It turns on if it's above the threshold and turns off if it's below, which is a simplified view of how neural networks work.
+
+In 1947, Donald Hebb had the idea that neurons in the brain learn by modifying the strength of the connections between neurons. This is called hyper learning, where if two neurons are fired together, then the connection linked between them increases; if they don't fire together, then the connection decreases.
+
+Later in 1948, cybernetics were proposed by Norbert Wiener, which is the idea that by having systems with sensors and actuators, you have a feedback loop and a self-regulatory system. The rules of the feedback mechanism of a car all come from this work.
+
+In 1957, Frank Rosenblatt proposed the Perceptron, which is a learning algorithm that modifies the weights of very simple neural nets.
+
+Overall, this idea of trying to build intellectual machines by simulating lots of neurons was born in 1940s, took off in 1950s, and completely died in late 1960s. The main reasons for the field dying off in 1960 are:
```

Comment:
Everything is in English here. What's going on?

Reply:
I only copied this contents and translating it.